src/webui/CMakeLists.txt

```diff
@@ -35,4 +35,4 @@ add_library(qbt_webui STATIC
 
 target_sources(qbt_webui INTERFACE www/webui.qrc)
 
-target_link_libraries(qbt_webui PRIVATE qbt_base)
+target_link_libraries(qbt_webui PRIVATE qbt_base PUBLIC Qt::Widgets)
```

Comment:
Why WebUI should depend on `Qt::Widgets`?

Reply:
Compiler has error of not finding `QApplication` header at: https://github.com/qbittorrent/qBittorrent/blob/2c8447853ba0855d32af7708670acd771b715591/src/app/application.h#L37